python/cugraph/cugraph/structure/graph_primtypes.pyx

```diff
@@ -16,61 +16,93 @@
 # cython: embedsignature = True
 # cython: language_level = 3
 
-import numpy as np
 from libc.stdint cimport uintptr_t
 from libcpp.utility cimport move
 
 from rmm.pylibrmm.device_buffer cimport DeviceBuffer
-from cudf.core.buffer import as_buffer
+import pylibcudf as plc
```

Comment:
This is minor but we've been using "plc" to mean "pylibcugraph" for 3 or more years within cugraph team and code, and I'm thinking this could be confusing. Can this be changed to "plcudf" or just kept as "pylibcudf"?

Reply:
Yup definitely, changed. (cuDF is starting to abbreviate pylibcudf as plc as well 😅 )